PalasoUIWindowsForms/Reporting/MemoryManagement.cs

```diff
@@ -43,14 +44,12 @@ public static bool CheckMemory(bool minor, string eventDescription, bool okToBot
 			string message;
 			ulong totalPhysicalMemory = 0;
 			string totalVirtualMemory = "unknown";
-			if (Platform.IsWindows)
-			{
+
+#if !__MonoCS__ // There are completely different dependencies and code per platform for finding the memory information
 				var computerInfo = new Computer().Info;
 				totalPhysicalMemory = computerInfo.TotalPhysicalMemory;
```

Comment:
One could fully specify Computer here and avoid the need for the using statement above.  Or one could leave well enough alone.  :-)